src/utils/sources-tree/getURL.js

```diff
@@ -54,13 +54,16 @@ function _getURL(source: Source, defaultDomain: string): ParsedURL {
         group: `${protocol}//${host || ""}`
       };
 
-    case "webpack:":
     case "ng:":
+    case "webpack:":
+      const path = pathname.substring(2);
```

Comment:
using whatwg-url will automatically do this. Can I introduce that module in src/utils/url.js ?

Reply:
Yea, we can try that